data/approvals.ts

```diff
@@ -0,0 +1,38 @@
+import qs from 'qs'
+import useSWR from 'swr'
+import { Approval } from '../types/interfaces'
+import { fetcher } from '../utils/fetcher'
+
+export type ApprovalCategory = 'Upload' | 'Deployment'
+export type ReviewFilterType = 'user' | 'archived'
```

Comment:
Should this be `ApprovalFilterType`?

Reply:
I've not refactored "review"s, only "request"s. Do you also want to change all reviews to approvals?